server/area_manager.py

```diff
@@ -410,8 +412,7 @@ def change_background(self, bg: str):
             Raises:
                 AreaError: if `bg` is not in background list
             """
-
-            if bg.lower() not in (name.lower()
+            if self.custom_bg == False or bg.lower() not in (name.lower()
```

Comment:
Will always say "invalid background name" if custom backgrounds is set to false.

```suggestion
            if not self.custom_bg and bg.lower() not in (name.lower()
```